flink-table/flink-table-planner-blink/src/main/java/org/apache/flink/table/planner/plan/nodes/exec/utils/ExecNodeUtil.java

```diff
@@ -42,8 +42,8 @@
      *
      * <p>TODO: This method can be removed once FLINK-20879 is finished.
      */
-    public static long getMemorySize(TableConfig tableConfig, ConfigOption<String> option) {
-        return MemorySize.parse(tableConfig.getConfiguration().getString(option)).getBytes();
+    public static long getMemorySize(TableConfig tableConfig, ConfigOption<MemorySize> option) {
```

Comment:
I think we don't need this utility method anymore.

Reply:
okey~